easybuild/scripts/install-easybuild.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+## Install EasyBuild and create a file "setup-env.sh" to use it
+
+# To use EasyBuild you will need a working modules tool in your machine (environment modules or Lmod)
+
+# Examples:
+#
+# To install latest stable release to /tmp/eb-stable
+# ./install-easybuild.sh /tmp/eb-stable stable
+#
+# To install latest devel release to /tmp/eb-devel
+# ./install-easybuild.sh /tmp/eb-devel develop
+
+if [ $# -ne 2 ]
+then
+    echo "Usage: $0 <full install path> <stable|develop>"
+    exit 1
+fi
+
+if [ "$2" != "stable" ] && [ "$2" != "develop" ]
+then
+    echo "Branch name must be stable or develop"
+    echo "Usage: $0 <full install path> <stable|develop>"
+    exit 1
+fi
+
+INSTALL_DIR=`readlink -f $1`
+BRANCH=$2
+LOGFILE=$INSTALL_DIR/install.log
+
+if [ "$2" == "stable" ]
+then
+    BRANCH="master"
+fi
+
+echo -e "starting installation. Logging to ${LOGFILE}\n"
+
+if [ ! -d "$INSTALL_DIR" ]; then
+    mkdir $INSTALL_DIR
+fi
+
+for repo in framework easyconfigs easyblocks;
+do
+    echo "downloading http://github.com/hpcugent/easybuild-${repo}/archive/${BRANCH}.tar.gz"
```

Comment:
set url in variabel, and use it to make sure this line and the next are the same url